python/apps/taiga/src/taiga/comments/mixins.py

```diff
@@ -9,11 +9,9 @@
 from taiga.comments.models import Comment
 
 
-class CommentsMixin(models.Model):
+class StoryCommentsMixin(models.Model):
     comments = models.GenericRelation(
-        Comment,
-        content_type_field="object_content_type",
-        object_id_field="object_id",
+        Comment, content_type_field="object_content_type", object_id_field="object_id", related_query_name="story"
```

Comment:
Use `related_query_name` is not a generic approach.

Reply:
You're right. I'tll try to find a generic approach for the mixin

Reply:
I'll remove the `related_query_name="story"` part